meinberlin/apps/plans/exports.py

```diff
@@ -66,6 +65,7 @@ def get_virtual_fields(self, virtual):
         virtual["projects_links"] = gettext("Project Links")
         virtual["created"] = gettext("Created")
         virtual["modified"] = gettext("Modified")
+        virtual["draft"] = gettext("Draft")
```

Comment:
Did you try to ust add it to the fields (line 25 - 39)? That should be enough, I think.

Reply:
@fuzzylogic2000 it actually works, but would use "is_draft" as column title. And it does not append to the end of the list. Shall I keep it as virtual field then?

Reply:
Ah no, I haven't seen that we do have wording here.